benchconnect/README.md

```diff
@@ -18,6 +18,28 @@ With [pipx](https://pypa.github.io/pipx/):
 pipx install benchconnect@git+https://github.com/conbench/conbench.git@main#subdirectory=benchconnect
 ```
 
+## Environment variables
+
+benchconnect relies on a number of environment variables to obtain various
+metadata:
+
+- `CONBENCH_URL`: The URL of the Conbench server. Required.
+- `CONBENCH_EMAIL`: The email to use for Conbench login. Only required if the
+server is private.
+- `CONBENCH_PASSWORD`: The password to use for Conbench login. Only required
+if the server is private.
+- `CONBENCH_REPOSITORY`: The URL of the repo being benchmarked. Defaults
+to `"https://github.com/apache/arrow"` if unset.
```

Comment:
Is this GitHub-specific, is this expected to start with `https://github.com`?

Reply:
Oops, these docs are slightly wrong; I copied them over from arrowbench, where defaults are arrow-specific. `CONBENCH_REPOSITORY` can be `org/repo` or the full URL; see https://github.com/conbench/conbench/blob/main/conbench/entities/run.py#L287-L293 I'll update this to use those descriptions instead so everything is consistent.